src/plugins/walletConnectToDapps/components/header/WalletConnectToDappsHeaderButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { ChevronDownIcon, ChevronRightIcon } from '@chakra-ui/icons'
+import { Menu, MenuButton, MenuList } from '@chakra-ui/menu'
+import { Button } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletConnectIcon } from 'components/Icons/WalletConnectIcon'
+import { RawText } from 'components/Text'
+
+import { DappAvatar } from './DappAvatar'
+import { DappHeaderMenuSummary } from './DappHeaderMenuSummary'
+
+export const WalletConnectToDappsHeaderButton = () => {
+  const dapp: any = {
```

Comment:
no any plz

Reply:
I know, this is just because it's dummy data and will be replaced by properly typed stuff later